tests/unit/broswe-storage-service.js

```diff
@@ -0,0 +1,34 @@
+describe('BrowserStorageService', function() {
+	var browserStorage;
+
+	beforeEach(module('ucApp'));
+
+	beforeEach(inject(function(_browserStorage_) {
+		// The injector unwraps the underscores (_) from around the parameter names when matching
+		browserStorage = _browserStorage_;
+	}));
+
+	describe('browserStorage', function() {
```

Comment:
Like the file. I don't use *should* any more as I find that `it('sets and retrieves a string');` reads just as well without the boiler plate. That's just personal preference though.